basics/getting_started.md

```diff
@@ -5,23 +5,30 @@ title: Getting Started
 
 # Getting started
 
-This page helps you set up your coding environment, describes creating an
-IntelliJ project and takes you through compiling and debugging your project.
-Finally, you can read about Bazel's source code, how to navigate through it, and
-how to use the Bazel CI system.
+This page covers how to start contributing code to Bazel. It covers how to set up
+your coding environment, describes creating an IntelliJ project and takes you
+through compiling and debugging your project.
+
+After you set up your environment, there is a quick overview of the structure of
+Bazel's code base, how to search and navigate the code, and how to monitor your
+builds with Bazel's continuous integration system.
 
 ## Installing Bazel
 
-*  [Install Bazel](https://bazel.build/versions/master/docs/install.html) on your
-   system. Note that you need the latest released version of Bazel for
-   development.
-*  Clone Bazel's Git repository from GitHub:
+Before you start developing, you'll need to
+
+1.  Install the latest version of Bazel on your system. For instructions, see
+    [Compiling Bazel from source](https://docs.bazel.build/versions/master/install-compile-source.html).
+
+2.  Clone Bazel's Git repository from GitHub:
```

Comment:
I realize I wrote the actual numbers for the list in my comments, but because I was adding a step, I wanted it to be clear where it would go. It's a best practice to use only "1." for ordered lists so when you add a list item the numbering automatically updates.

Reply:
Done.